### DISCUSSION TITLE
Quick instance switching

### DISCUSSION THREAD

**Discussion by Magess** [NickColley/semaphore#35] [New features and ideas]
I would love to have something in the top nav bar that would let me switch instances rather than having to go into Settings > Instances and click on the star.

There's still a lot of unused real estate in the nav bar. Having the name of the instance appear there (or maybe a selectable icon if you want to conserve space... Server 1 = 🟪, Server 2 = 🔺) with a dropdown box if you click on it to switch would be much more convenient.

**Comment by NCW-Srandall** [comment]
Of course, a keyboard shortcut would be important here too. :)
